### DISCUSSION TITLE
Human Feedback

### DISCUSSION THREAD

**Discussion by jeanesh** [StreetLamb/tribe#105] [General]
Amazing work!

How best should I implement a human feedback step? It looks like this could be achieved either by adding a human node or doing something similar to the interrupt. Another option is I create a tool to take the feedback but then that would bypass the chat.

What would you recommend as an approach? Also, are you considering extending this project to support this option in the future?

Thanks

J

**Comment by StreetLamb** [comment]
Hi @jeanesh , this is something I am currently tackling! How I envision it is to have an additional Human-In-The-Loop feature similar to interrupt. If activated on a node, the node will wait for human output instead getting the output from a LLM.

**Reply by jeanesh** [reply]
Ok, brilliant! Looking forwards to it! Thanks
